### PR TITLE
HZN-1433: Update integration api to handle alarm feedback without Kafka

### PR DESCRIPTION
**Pull request by mattixtech**
This PR adds support for consuming alarm feedback from a direct data source without Kafka.

### CODE REVIEW

**Comment by mattixtech** [comment]
Note: haven't manually tested this end to end yet.

**Comment by mattixtech** [comment]
Tested alarm feedback via the integration API. It is working now with my latest changes.

**Comment by mattixtech** [comment]
I'm assuming the build failure will be resolved when the API changes are merged.

File: karaf-features/src/main/resources/features.xml [hunk 1]
```diff
@@ -13,6 +13,11 @@
         <feature dependency="true">aries-blueprint</feature>
     </feature>
 
+    <feature name="oce-datasource-common" description="OCE :: Datasource :: Common" version="${project.version}">
+        <bundle>mvn:org.opennms.oce.datasource/org.opennms.oce.datasource.opennms-common/${project.version}</bundle>
```
Comment (j-white) [inline karaf-features/src/main/resources/features.xml]: This bundle doesn't appear to exist here.

File: karaf-features/src/main/resources/features.xml [hunk 2]
```diff
@@ -13,6 +13,11 @@
         <feature dependency="true">aries-blueprint</feature>
     </feature>
 
+    <feature name="oce-datasource-common" description="OCE :: Datasource :: Common" version="${project.version}">
```
Comment (j-white) [inline karaf-features/src/main/resources/features.xml]: Should this be `oce-datasource-opennms-common`?

**Comment by j-white** [comment]
Build is still failing after a re-run, since these changes a *direct*ly related to those in the PR for HZN-1441 it may be worth just combining them together in a single branch.

**Comment by mattixtech** [comment]
Closing, see #52. Comments will be addressed there.

**Pull request closed without merging**